python/lsst/obs/base/gen3/ingest.py

```diff
@@ -144,6 +161,17 @@ def __init__(self, config=None, *, butler, **kwds):
         # that lose conflicts.  Note that this doesn't actually add this Run
         # to the Registry; we only do that on first use.
         self.stashRun = Run(self.config.stash) if self.config.stash is not None else None
+        self.visitRegions = {}
+
+    def addVisitRegions(self):
```

Comment:
What is a Visit Region? (I'm still concerned we don't know what "visit" means in general).

Reply:
It's a `sphgeom.ConvexPolygon` covering the full-focal-plane region of a visit.

Reply:
Ok, firstly I think I'm asking for a docstring to explain what's happening. The second comment is about the use of the word "visit" and worrying about naming...

Reply:
I don't think I'm going to get this in the weekly tonight, so I'll let @natelust handle the docstring request.  As for second part, "visit" is definitely the right terminology for the current schema and concrete dimensions, as that's the table that actually has a region field (because - at least at present - a visit is a combination of exposures that is definitely at the same position on the sky, while an exposure isn't necessarily on the sky at all).  That may change in the future, but I'm not certain it will even if we introduce other kinds of groups of exposures.